apps/src/package.json

```diff
@@ -11,7 +11,6 @@
     ]
   },
   "browserify-shim": {
-    "marked": "global:marked",
-    "radium": "global:Radium"
+    "marked": "global:marked"
```

Comment:
This was a little tricky; the `browserify-shim` entry for Radium here is now redundant with the `browserify-global-shim` entry in apps/package.json, and this was shimming Radium in tests when we don't want to anymore.